src/service/performance-impl.js

```diff
@@ -715,30 +715,27 @@ export class Performance {
    *     this directly.
    */
   tick(label, opt_delta) {
-    const value = opt_delta == undefined ? this.win.Date.now() : undefined;
+    let data;
+    let storedVal;
+
+    // Absolute value case (not delta).
+    if (opt_delta == undefined) {
+      // Marking only makes sense for non-deltas.
+      this.mark(label);
+      const now = this.win.Date.now();
+      data = dict({'label': label, 'value': now});
+      storedVal = now - this.initTime_;
+    } else {
+      data = dict({'label': label, 'delta': Math.max(opt_delta, 0)});
```

Comment:
Does this mean we no longer send `data.value` to `sendMessage()` and `queueTick_()`?

Reply:
Correct. Isn't that better?

Reply:
Oh I see, lol. Thanks for cleaning this up.

Minor suggestion:
```js
const data = dict({'label': label});
if (...) {
  data['value'] = now;
} else {
  data['delta'] = Math.max(opt_delta, 0);
}
```